docs/docs/deploy/clients.md

```diff
@@ -3,6 +3,9 @@ layout: default
 title: Install Clients
 nav_order: 5
 parent: Deploy
+description: >
+  Firezone is compatible the official WireGuard clients available
```

Comment:
typo: compatible *with*

Reply:
& missing a period at the end

Reply:
the sentence continues after this line with a period.